README.md

```diff
@@ -28,14 +28,20 @@ Environment variable values can be found by asking a member of Cornell AppDev.
 
 #### Development
 
-Run the following:
+To run ithaca-transit-backend locally, run the following:
 
-```
+```bash
 npm run build:dev
 npm run start:microservices
 npm run start:node
 ```
 
+```build:dev``` builds node, ```start:microservices``` starts up its microservices, and ```start:node``` runs node. 
+
+You only need to run ```npm run build:dev``` and ```npm run start:microservices``` once - the "volumes" part of docker-compose.yml allows for hot reloading. However, if you are building an image for production, make sure to do ```npm run build:dev``` right before doing that. And you will always need to run ```npm run start:node``` to run node locally.
+
+> It is helpful to understand the scripts in package.json. When you run ```npm run start:node```, it goes to find start:node's value in package.json's scripts. You can see that it's ```docker-compose up node```. This makes sense because in docker-compose.yml, you can see that the target is development. The command for starting development is ```npm run start:dev```; in package.json, its value tells us to actually execute some magic (./node_modules/.bin/webpack…). The reason why we do this is because ```npm run start:node``` lets us run on our local computers while ```npm run start:dev``` runs inside the image.
```

Comment:
Maybe we can make this section more succinct! You could do something like
```suggestion
> When you run ```npm run start:node```, it actually runs ```docker-compose up node```.  On the other hand, the command for starting development is ```npm run start:dev```; in package.json, its value tells us to actually execute some magic (./node_modules/.bin/webpack…). The main difference between these two is that ```npm run start:node``` lets us run node inside the image while ```npm run start:dev``` runs on our local computer.
```
@alanna-zhou can you read this again

Reply:
That looks better. We might still want to reference package.json in the first line, depending on whether the prospective viewer will already know to look there.

Reply:
Cool sounds like a good idea!